### PR TITLE
Use Proxy for GSFont.features/featurePrefixes

### PR DESCRIPTION
**Pull request by khaledhosny**
Matches Glyphs App API and for consistency with the rather identical GSFont.classes.

### CODE REVIEW

File: tests/classes_test.py [hunk 1]
```diff
@@ -1749,5 +1749,23 @@ def test_remove_glyphs(self):
             del self.font.glyphs[self.font]
 
 
+class FontClassesProxyTest(unittest.TestCase):
+    def setUp(self):
+        self.font = GSFont(TESTFILE_PATH)
+
+    def test_remove_glyphs(self):
```
Comment (schriftgestalt) [inline tests/classes_test.py:1756]: Is this a good name for the test?

**Comment by madig** [comment]
Out of curiosity, is there something that didn't work before but does with this PR?

**Comment by khaledhosny** [comment]
> Out of curiosity, is there something that didn't work before but does with this PR?

Without this PR they are simple lists, so you can’t access them by name, like in the examples in https://docu.glyphsapp.com/#GSFont.features. I was merging two fonts and trying to merge `Languagesystems` prefix instead of having two of it when I found this limitation. It can be worked around of course.

File: tests/classes_test.py (hunk 1 above)
Comment (khaledhosny) [reply tests/classes_test.py]: Sorry, copy&paste.

**Comment by anthrotype** [comment]
thanks

**Pull request merged**